src/machine/behavior/is/memory/ldbu.rs

```diff
@@ -1,6 +1,14 @@
 use machine::state::State;
+use machine::state::mem::ByteAt;
 
-pub fn ldbu(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
-}
+pub fn ldbu(state: &mut State, x: u8, y: u8, z: u8) {
+
+    let op1: u64 = state.gpr[y].into();
+    let op2: u64 = state.gpr[z].into();
+    let mut a = op1 + op2;
+    a = a % 2u64.pow(64);
```

Comment:
You might want to use `overflowing_add()` or a similar method of `u64` here to clearly express your intend in the code.